tests/handlers/test_multi_image_input_adapter.py

```diff
@@ -0,0 +1,101 @@
+import mimetypes
+import string
+import random
+
+from werkzeug import Request
+
+from bentoml.adapters.multi_image_input import MultiImageInput
+from bentoml.marshal.utils import SimpleRequest
+
+
+def generate_multipart_body(image_file):
```

Comment:
Might be best to just use requests' multipart encode feature:

https://requests.readthedocs.io/en/master/user/advanced/#post-multiple-multipart-encoded-files